pkg/apis/azureprovider/v1alpha1/types.go

```diff
@@ -65,12 +65,11 @@ type AzureMachineProviderCondition struct {
 	Message string `json:"message"`
 }
 
-type MachineRole string
-
 const (
-	// TODO: Change references to "controlplane", instead of "Master" and lowercase both roles.
-	Master MachineRole = "Master"
-	Node   MachineRole = "Node"
+	// ControlPlane machine label
+	ControlPlane string = "controlplane"
```

Comment:
This change was due to linter complaining use of "controlplane" and "node" duplication across projects